server/src/interfaces.ts

```diff
@@ -61,26 +61,42 @@ export interface CommitComputedProperties {
 
 /**
  * CommitRequest Interface that contains the fields that will be provided
- * by the client in the POST/PUT request body.
+ * by the client in the request body to create a Commit.
  */
 export interface CommitRequest {
   customerId: number;
   listingId: number;
 }
 
+/**
+ * CommitPaymentRequest Interface that contains the fields that will be provided
+ * by the client in the request body.
+ */
+export interface CommitPaymentRequest {
+  deliveryAddress: string;
```

Comment:
maybe should wrap these kind of information in a container called "FulfilmentDetails", which sounds more like a service waiting to be fulfilled.

Reply:
@xlai20 
Hmm what do you mean by that? `CommitPaymentRequest` is the type of the payment request body.

Did you mean having a separate `FulfilmentDetails` type? Meaning doing something like:
```
interface FulfilmentDetails {
  deliveryAddress: string;
  deliveryContactNumber: string;
}

export interface CommitPaymentRequest {
  fulfilmentDetails: FulfilmentDetails;
}
```

Reply:
yes, that was what I meant.